pod/perlgov.pod

```diff
@@ -479,7 +479,7 @@ Foundation will select a Vote Administrator.
 
 =over
 
-=item * Neil Bowers
+=item * Philippe "BooK" Bruhat
```

Comment:
Tiny nit-pick: are we maintaining these in alphabetical order? The last list was.